release-notes/8.0/preview/api-diff/preview4/Microsoft.NETCore.App/8.0-preview4_System.Runtime.CompilerServices.md

```diff
@@ -0,0 +1,13 @@
+# System.Runtime.CompilerServices
+
+``` diff
+ namespace System.Runtime.CompilerServices {
+     public sealed class RuntimeWrappedException : Exception {
+         public override void GetObjectData(SerializationInfo info, StreamingContext context);
+     }
+     public sealed class SwitchExpressionException : InvalidOperationException {
+         public override void GetObjectData(SerializationInfo info, StreamingContext context);
+     }
+ }
+```
+
```

Comment:
Did something change with the tooling? We have a lot of changes over `GetObjectData`, `ISerializable`, and related bits that haven't actually changed AFAIK.

Reply:
Yes adding serialization methods and construction is creating a lot of noise in the diff.

Reply:
We also saw this in System.Diagnostics.Tracing